pandas/core/arrays/arrow/array.py

```diff
@@ -581,6 +572,122 @@ def isna(self) -> npt.NDArray[np.bool_]:
 
         return self._data.is_null().to_numpy()
 
+    def any(self, *, skipna: bool = True, **kwargs):
```

Comment:
I guess technically `ArrowExtensionArray` should subclass `ExtensionArraySupportsAnyAll` now instead?